addon/instance-initializers/prefetch.js

```diff
@@ -41,7 +42,7 @@ export function initialize(instance) {
       // Build fullParams as in unresolved-handler-info-by-param#getModel.
       let fullParams = handlerInfo.params || {};
       if (transition && transition.queryParams) {
-        fullParams = Ember.copy(fullParams);
+        fullParams = copy(fullParams);
```

Comment:
Hmm, since we only need a shallow clone here, can we use `Ember.assign` instead?

Reply:
So it looks like this Ember try scenario goes back to 2.4 which does not have Ember.assign and so we get `Ember.assign is not a function`. Would `Object.assign` work just as well?

Reply:
@brandynbennett do this instead:

```
import { assign, merge } from '@ember/polyfills';
...
fullParams = (assign || merge)(full Params);
```